terraform/azure_rke2_cluster/variables.tf

```diff
@@ -95,6 +95,6 @@ variable "address_space" {
 
 variable "ssh_public_key_path" {
   type        = string
-  description = "The path to a public SSH key to be mounted on hosts."
+  description = "The path to a public SSH key to be mounted on hosts. Windows requires a rsa ssh key pair."
```

Comment:
Could this be changed to `This must be a RSA public key.`? The field is specific to the public key